langkit/templates/pkg_analysis_spec_ada.mako

```diff
@@ -335,6 +335,10 @@ package ${ada_lib_name}.Analysis is
    procedure PP_Trivia (Unit : Analysis_Unit'Class);
    --  Debug helper: output a minimal AST with mixed trivias
 
+   function Get_Line
```

Comment:
What about adding the `overriding` keyword to make it clear this is supposed to override the text_buffer_ifc primitive?